### PR TITLE
Closes #788: Migration for D7 well paragraph type (to cards).

### LINKED ISSUE #788: Migrate Well paragraph type
See migration mapping doc for details: https://docs.google.com/spreadsheets/d/134UMiYtE8f1YAKYgSukL2JxFzDJF6-TgCAvYFC-vxqQ/edit#gid=0

### PR DESCRIPTION
**Pull request by smitahantzios**
Paragraph Migration for Well to AZ Cards

## Description
Paragraph Migration for Well to AZ Cards

## Related Issue

Closes #788 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I've added this Pull Request to the AZ Quickstart Github project and set it to "Review in progress".
- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: modules/custom/az_migration/migrations/migrate_plus.migration.az_paragraph_well.yml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    source: field_uaqs_summary
+    index:
+      - 0
+      - value
+
+  field_az_cards/body_format:  
+    plugin: default_value
+    default_value: az_standard
+    
+  behavior_settings:
```
Comment (trackleft) [inline modules/custom/az_migration/migrations/migrate_plus.migration.az_paragraph_well.yml:29]: This should be the same as the single card, correct?

Comment (smitahantzios) [reply modules/custom/az_migration/migrations/migrate_plus.migration.az_paragraph_well.yml:29]: @trackleft thats correct!

**Comment by danahertzberg** [comment]
I updated the [spreadsheet](https://docs.google.com/spreadsheets/d/134UMiYtE8f1YAKYgSukL2JxFzDJF6-TgCAvYFC-vxqQ/edit#gid=0) for Well starting on row 94. The selections should change for card style and card background.

**Comment by smitahantzios** [comment]
> I updated the [spreadsheet](https://docs.google.com/spreadsheets/d/134UMiYtE8f1YAKYgSukL2JxFzDJF6-TgCAvYFC-vxqQ/edit#gid=0) for Well starting on row 94. The selections should change for card style and card background.

Done

**Review by danahertzberg — DISMISSED** [review]
Looks good from a requirements perspective

**Review by tadean — DISMISSED** [review]

**Review by trackleft — APPROVED** [review]

**Review by tadean — APPROVED** [review]

**Review by danahertzberg — APPROVED** [review]

**Pull request merged**